winbuild/build.rst

```diff
@@ -87,11 +87,18 @@ are set by running ``winbuild\build\build_env.cmd`` and install Pillow with pip:
     winbuild\build\build_env.cmd
     python.exe -m pip install -v -C raqm=vendor -C fribidi=vendor .
 
+You can also install Pillow in `editable mode`_::
+
+    winbuild\build\build_env.cmd
+    python.exe -m pip install -v -C raqm=vendor -C fribidi=vendor -e .
+
 To build a wheel instead, run::
```

Comment:
```suggestion
To build a non-editable wheel instead, run::
```

(the editable hook does produce special "editable" wheels too, under the hood)

Reply:
I would expect anyone who understands the distinction between regular and editable wheels can figure that out from the command, so I'm not sure how helpful this change would be. Personally, I think "non-editable" is more confusing than helpful.

Would "binary" make sense here?
```suggestion
To build a binary wheel instead, run::
```